docs/rpc-endpoints.md

```diff
@@ -540,3 +495,41 @@ Error examples:
   "reason_code": "ChainstateError"
 }
 ```
+
+### GET /v3/blocks/[Block ID]
+
+Fetch a Nakamoto block given its block ID hash.  This returns the raw block
+data.
+
+This will return 404 if the block does not exist.
+
+### GET /v3/tenures/[Block ID]
+
+Fetch a Nakamoto block and all of its ancestors in the same tenure, given its
+block ID hash.  At most `MAX_MESSAGE_LEN` (i.e. 2 MB) of data will be returned.
+If the tenure is larger than this, then the caller can page through the tenure
+by repeatedly invoking this endpoint with the deepest block's block ID until
+only a single block is returned (i.e. the tenure-start block).
+
+This method returns one or more raw blocks, concatenated together.
+
+This method returns 404 if there are no blocks with the given block ID.
+
+### GET /v3/tenures/info
+
+Return metadata about the highest-known tenure, as the following JSON structure:
+
+```json
+{
+  "consensus_hash": "dca60a97a135189d67a5ad6d2dac90f289b19c96",
+  "reward_cycle": 5,
+  "tip_block_id": "317c0ee162d1ee02c67d5bca79003dafc59aa84579360387f43650c37491ac3b",
+  "tip_height": 116
+}
+```
+
+Here, `consensus_hash` identifies the highest-known tenure (which may not be the
+highest sortition), `reward_cycle` identifies the reward cycle number of this
+tenure, `tip_block_id` idenitifies the highest-known block in this tenure, and
+`tip_height` identifies that block's height.
+
```

Comment:
Use `docs/openapi.yml` instead. `rpc-endpoints.md` should really just be removed.

Reply:
Done